### DISCUSSION TITLE
Owner and permissions not observed, need to add -map.uid?

### DISCUSSION THREAD

**Discussion by gqqnbig** [seaweedfs/seaweedfs#3125] [Q&A]
```console
$ whoami
gqqnbig
$ echo hello > f
$ cat f
hello
$ sudo su lyi
$ ll
total 1
drwxr-xr-x 1 gqqnbig gqqnbig 0 May 31 15:15 ./
drwxr-xr-x 1 root  root  0 May 31 15:15 ../
-rw-rw-r-- 1 gqqnbig gqqnbig 6 May 31 15:15 f
$ rm f
$ ll
total 0
drwxr-xr-x 1 gqqnbig gqqnbig 0 May 31 15:15 ./
drwxr-xr-x 1 root  root  0 May 31 15:15 ../
```
I created file f as gqqnbig. Then I log in as lyi. Although f is read-only for lyi, lyi is able to delete this file.

From https://github.com/chrislusf/seaweedfs/issues/2275, I guess it's related to -map.gid and -map.uid. But how do I pass in the parameter? Can you give an example? In #2275, the right hand size of `<local_uid>:<filer_uid>` is all 0. 

Also, my server has 50 users, do I have to list all 50 uids and uids?

**Comment by chrislusf** [comment]
Yes. See `weed mount -h`.
